Doc/library/inspect.rst

```diff
@@ -693,6 +693,10 @@ function.
    .. versionchanged:: 3.10
       The *globals*, *locals*, and *eval_str* parameters were added.
 
+   .. versionchanged:: 3.13
+      The *skip_bound_arg* parameter was added.
+      Pass ``False`` to keep the ``self`` parameter in a signature.
```

Comment:
Please move this sentence ("Pass...") from this note to the doc body. Or rephrase it to something like:

If *skip_bound_arg* is false, keep `self` parameter in the signature.